api/build.gradle

```diff
@@ -1,47 +1,10 @@
-plugins {
-    id "com.github.johnrengelman.shadow" version "7.1.1"
-    id 'java-library'
-    id 'maven-publish'
-}
 repositories {
     mavenLocal()
     mavenCentral()
 }
 
 dependencies {
-    implementation "com.googlecode.json-simple:json-simple:$jsonSimpleVersion"
-    implementation "com.google.guava:guava:$guavaVersion"
-    compileOnly "org.slf4j:slf4j-api:1.7.30"
-}
-
-ext.artifactName = 'platform-abstraction-layer-api'
-ext.artifactVersion = project.version
-
-publishing {
```

Comment:
I can't publish the api module if this is removed - as far as I know anything that declares a compile time dependency still downloads the required artifact from maven - a dependency tree if you will

Reply:
It's not gone - it's just taken out from submodules and put into the main build.gradle since it was duplicated code in every module

Reply:
Oh it's in subprojects now, my bad (Don't review on mobile kids)